app/src/main/java/com/kks/nimblesurveyjetpackcompose/model/ApiInterface.kt

```diff
@@ -0,0 +1,13 @@
+package com.kks.nimblesurveyjetpackcompose.model
+
+import com.kks.nimblesurveyjetpackcompose.model.request.LoginRequest
+import com.kks.nimblesurveyjetpackcompose.model.response.BaseResponse
+import com.kks.nimblesurveyjetpackcompose.model.response.LoginResponse
+import retrofit2.Call
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+interface ApiInterface {
+    @POST("api/v1/oauth/token")
+    fun loginUser(@Body loginRequest: LoginRequest): Call<BaseResponse<LoginResponse>>
```

Comment:
Since we're using coroutines, why not make these into suspend functions and get rid of the `Call`? 🤔